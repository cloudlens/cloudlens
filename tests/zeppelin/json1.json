{"paragraphs":[{"text":"%cl\nsource(\"local:tests/zeppelin/json1.log\", json, obj.field.array)","dateUpdated":"Jul 2, 2016 10:16:06 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467512166773_-798495757","id":"20160702-221606_1688366694","result":{"code":"SUCCESS","type":"TABLE","msg":"\n%html <font color=blue>hi</font>:&quot;Bonjour&quot;\t%html <font color=blue>message</font>:&quot;6 pommes&quot;\t%html <font color=blue>foo</font>:{&quot;a&quot;:3}\t\n%html <font color=blue>hi</font>:&quot;Hello&quot;\t%html <font color=blue>message</font>:&quot;2 apples&quot;\t%html <font color=blue>foo</font>:{&quot;a&quot;:{&quot;b&quot;:5}}\t\n%html <font color=blue>test</font>:42\t","comment":"","msgTable":[[{"value":"%html <font color=blue>hi</font>:&quot;Bonjour&quot;"},{"value":"%html <font color=blue>message</font>:&quot;6 pommes&quot;"},{"value":"%html <font color=blue>foo</font>:{&quot;a&quot;:3}"},{"value":""}],[{"value":"%html <font color=blue>hi</font>:&quot;Hello&quot;"},{"value":"%html <font color=blue>message</font>:&quot;2 apples&quot;"},{"value":"%html <font color=blue>foo</font>:{&quot;a&quot;:{&quot;b&quot;:5}}"},{"value":""}],[{"value":"%html <font color=blue>test</font>:42"},{"value":""}]],"columnNames":[],"rows":[["%html <font color=blue>hi</font>:&quot;Bonjour&quot;","%html <font color=blue>message</font>:&quot;6 pommes&quot;","%html <font color=blue>foo</font>:{&quot;a&quot;:3}",""],["%html <font color=blue>hi</font>:&quot;Hello&quot;","%html <font color=blue>message</font>:&quot;2 apples&quot;","%html <font color=blue>foo</font>:{&quot;a&quot;:{&quot;b&quot;:5}}",""],["%html <font color=blue>test</font>:42",""]]},"dateCreated":"Jul 2, 2016 10:16:06 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:1613"},{"text":"%cl\n{\n    print(\"init\")\n}\n\nmatch upon (entry.hi){\n    \"(?<french>Bonjour)\"\n}\n\nstream{\n    print(entry.message)\n}\n\nstream(entry) when (entry.foo.a.b){\n    entry.foo.a.c = \"4012\";\n    print(entry.foo.a.b)\n}\n\n{\n    print(\"reset\")\n}\n\nstream{\n    print(entry.foo.a.c)\n}\n\nstream when (entry.french){\n    print(\"'\" + entry.message + \"' is in french.\")\n}\n\n{\n    print(\"post\")\n}","dateUpdated":"Jul 2, 2016 10:16:26 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467512166773_-798495757","id":"20160702-221606_1392493538","result":{"code":"SUCCESS","type":"TEXT","msg":"init\n6 pommes\n2 apples\n5\nreset\n'6 pommes' is in french.\n4012\npost\n"},"dateCreated":"Jul 2, 2016 10:16:06 PM","status":"READY","errorMessage":"","progressUpdateIntervalMs":500,"$$hashKey":"object:1614"},{"dateUpdated":"Jul 2, 2016 10:16:06 PM","config":{"colWidth":12,"graph":{"mode":"table","height":300,"optionOpen":false,"keys":[],"values":[],"groups":[],"scatter":{}},"enabled":true,"editorMode":"ace/mode/scala"},"settings":{"params":{},"forms":{}},"jobName":"paragraph_1467512166773_-798495757","id":"20160702-221606_779555140","dateCreated":"Jul 2, 2016 10:16:06 PM","status":"READY","progressUpdateIntervalMs":500,"$$hashKey":"object:1615"}],"name":"foo","id":"2BRFJ826X","angularObjects":{"2BRB4ZK9Z":[],"2BN98KZFM":[]},"config":{"looknfeel":"default"},"info":{}}
