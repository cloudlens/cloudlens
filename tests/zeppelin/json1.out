init
6 pommes
2 apples
5
reset
'6 pommes' is in french.
4012
post
